<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseDialog</class>
 <widget class="QDialog" name="CourseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>课程名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDay">
       <property name="text">
        <string>星期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboDay"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelStart">
       <property name="text">
        <string>开始节次</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboStart"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEnd">
       <property name="text">
        <string>结束节次</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboEnd"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelClassroom">
       <property name="text">
        <string>教室</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editClassroom"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelTeacher">
       <property name="text">
        <string>教师</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editTeacher"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelNote">
       <property name="text">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="editNote"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelColor">
       <property name="text">
        <string>颜色</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="buttonColor"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>